test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -1,5 +1,7 @@
 using namespace System.Diagnostics
 
+Import-Module $PSScriptRoot\..\Common\Test.Helpers.psm1
```

Comment:
Maybe it's just me noticing the carryover from sharing 1 test tree for many products, I'd just prefer a *flatter* structure and/or one that is easier to find corresponding tests, maybe something like:

```
test
  | Tools
    | Modules
      | TestHelper.psm1
  | System.Management.Automation
    | Microsoft.PowerShell.Commands
      | Get-Variable.tests.ps1
    | Language (or System.Management.Automation.Language)
      | Parser.tests.ps1
    | Remoting
      | Serialization.tests.ps1   
  | Microsoft.PowerShell.Commands.Utility
    | Object (grouping by noun, grouping by verb might make sense too)
      | Measure-Object.tests.ps1
      | New-Object.tests.ps1
  | Microsoft.PowerShell.Commands.Management
  | PSReadline
etc.
```

I realize some tests don't fall neatly into testing an api - and at least `System.Management.Automation` namespace is overly large, so requires some additional structure, but this gives an overview on what I'd find more intuitive.